src/pages/Explore/TraceExploration.tsx

```diff
@@ -307,6 +308,31 @@ export class TraceExplorationScene extends SceneObjectBase {
   };
 }
 
+const useServiceName = (model: SceneObject) => {
```

Comment:
Are there any plans to use the EntityAssertionsWidget in other places? In theory the logic to get service name could be encapsulated in that component (though it would require passing `model` or making it a scene object).